registry-photon/v2.10.2/rockcraft.yaml

```diff
@@ -0,0 +1,72 @@
+name: registry
+summary: Rock replacement for the Harbor registry image.
+description: >
+  This rock is a drop in replacement for the
+  docker.io/goharbor/registry-photon:v2.10.2 image.
+# Based on the following:
+# https://github.com/goharbor/harbor/tree/v2.10.2/make/photon/registry
+version: v2.10.2
+license: Apache-2.0
+
+base: ubuntu@22.04
+build-base: ubuntu@22.04
+platforms:
+  amd64:
+  arm64:
+
+services:
+  registry:
+    command: /home/harbor/entrypoint.sh
+    override: replace
+    startup: enabled
+    user: harbor
+    group: harbor
+    working-dir: /home/harbor
+
+parts:
+  create-harbor-user:
+    plugin: nil
+    overlay-script: |
+      groupadd -R $CRAFT_OVERLAY -r -g 10000 harbor
+      useradd -R $CRAFT_OVERLAY \
+        --no-log-init -r -m -g 10000 -u 10000 harbor
+
+  registry:
```

Comment:
FYI, [the `registryctl` image embeds the `registry` binary](https://github.com/goharbor/harbor/blob/v2.10.2/make/photon/registryctl/Dockerfile#L12) too for some reason so I'm just gonna blindly copy this part from you 👼